geode-core/src/main/java/org/apache/geode/internal/metrics/GeodeCompositeMeterRegistry.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.internal.metrics;
+
+import java.util.Set;
+
+import io.micrometer.core.instrument.binder.MeterBinder;
+import io.micrometer.core.instrument.composite.CompositeMeterRegistry;
+
+public class GeodeCompositeMeterRegistry extends CompositeMeterRegistry {
+
+  private final Set<MeterBinder> binders;
+
+  public GeodeCompositeMeterRegistry(Set<MeterBinder> binders) {
```

Comment:
I think it would be more standard to make `binders` and `this.binders` a `List<MeterBinder>` since we're only ever iterating over them and not doing add/remove/contains operations.

Reply:
Or perhaps `Iterable<MeterBinder>`

Reply:
I thought about using a List. Conferring with Dale he suggested a Set. I think both are sufficient.  Set provides protection against duplicates which concerned Dale, but not me in the context(rightly or wrongly so). Hence, why I went with the Set.

Reply:
> Or perhaps `Iterable<MeterBinder>`

`Iterable` makes it hard to create a stream to process the elements.